src/nvim/tui/tui.c

```diff
@@ -131,7 +131,7 @@ static void terminfo_start(UI *ui)
   data->can_use_terminal_scroll = true;
   data->bufpos = 0;
   data->bufsize = sizeof(data->buf) - CNORM_COMMAND_MAX_SIZE;
-  data->showing_mode = 0;
+  data->showing_shape = SHAPE_IDX_N;
```

Comment:
In fact the new "shape" index is _a lot_ more general (if we consider actual information content) than the old "mode" string. But I will happy to rename it to some thing more general sounding than `shape`.